daffodil-cli/src/it/scala/org/apache/daffodil/tunables/TestCLITunables2.scala

```diff
@@ -107,8 +106,8 @@ class TestCLITunables2 {
       } finally {
         shell.close()
       }
-    } while (false) // workaround scala local block bug.
-    do {
```

Comment:
Since you're making this change, can you also update to use `expectIn` instead of `expect` so we can remove the redirect? My concern is redirecting the input is going to cause stderr and stdout mixed up and might cause random failures when expecting strings depending on how things get mixed up. Might not actually be possible, but it's maybe good practice to avoid the redirect in favor of expectIn.

Reply:
Ok. 
Interestingly, fixing this do-while thing..... failed to compile on the github automated CI. Compiles on my machine tho.

Reply:
I was playing around with a small example, and for some reason I needed to provide the type on some variables. Might be the case here as well.